### ISSUE TITLE
Search box input element fails validation

### ISSUE THREAD

**Issue by jbickar** [SU-SWS/open_framework#115]
The validator at http://validator.w3.org/nu/ is complaining about two attributes in the search box input element.
1. The "value" attribute is not allowed on an input element
2. An "alt" attribute is required on an input element

http://validator.w3.org/nu/?schema=http%3A%2F%2Fs.validator.nu%2Fw3c-html5-microdata-rdfa.rnc+http%3A%2F%2Fs.validator.nu%2Fhtml5%2Fassertions.sch+http%3A%2F%2Fc.validator.nu%2Fbase%2F+http%3A%2F%2Fc.validator.nu%2Fmicrodata%2F&doc=https%3A%2F%2Fsites.stanford.edu%2Fundergrad

This is for input#edit-search-block-form--2, which you can see on https://sites.stanford.edu/undergrad/

**Comment by jbickar** [comment]
Also, the accessibility checker at achecker.ca is indicating that the "label text is empty". Presumably there should be a &lt;label&gt; element on the input.btn-search form-submit element:

``` html
<div class="form-actions form-wrapper" id="edit-actions"><input class="btn-search form-submit" type="image" id="edit-submit" name="submit" value="Search" src="https://sites.stanford.edu/adminguide/sites/all/themes/open_framework/images/searchbutton.png" /></div><input type="hidden" name="form_build_id" value="form-VnQCMSaYrVDkrv_6ZmUOb0dL_-98bi6V79XORP52D4Y" />
```

**Comment by jbickar** [comment]
Add this to open_framework_form_alter() :

$form['actions']['submit']['#attributes']['alt'] = t('Search button');

**Comment by jbickar** [comment]
Working on the error with the "value" attribute; it's a little trickier

**Comment by meganem** [comment]
Related #161
